### ISSUE TITLE
CRITICAL: Fix attachment generating properties file incorrectly.

### ISSUE THREAD

**Issue by TheElementalOfDestruction** [Syncurity/msg-extractor#26]
Lines:
```
self.__props = Properties(
    self.msg._getStream(self.msg.prefixList + [self.__dir, '__properties_version1.0']),
    constants.TYPE_ATTACHMENT)
```
in `attachment.Attachment.__init__` should be:
```
self.__props = Properties(
    self.msg._getStream([self.__dir, '__properties_version1.0']),
    constants.TYPE_ATTACHMENT)
```

**Comment by punkrokk** [comment]
@TheElementalOfCreation So this should get changed on next release? and have a release done?
 Did this used to work before?

**Comment by TheElementalOfDestruction** [comment]
It used to SOMEHOW work in most cases, but it should not have. I'm pretty sure it was a combination of luck and not testing the right files (embedded msg) caused by a relatively recent change. (This issue was not present to my knowledge in 0.20.0).

The next release we have needs to include the fix, but let's finish our stuff before that release is public

**Comment by TheElementalOfDestruction** [comment]
Specifically, we want it to be fixed by version 0.21.0

**Comment by punkrokk** [comment]
@TheElementalOfCreation you had me paste this in my branch?
